index.html

```diff
@@ -0,0 +1,167 @@
+<!-- 
+    Style: https://www.touchbistro.com/features/restaurant-inventory-management/
+    Bootstrap Jumbotron: https://getbootstrap.com/docs/4.3/examples/jumbotron/
+    Content: see style
+    Images: find source it was in a chat somewhere
+ -->
+
+<!doctype html>
+<html lang="en">
+
+<head>
+    <title>Soup Marketing Page</title>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=yes">
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO"
+        crossorigin="anonymous">
+    <link href="css/index.css" rel="stylesheet">
+    <!-- Optional JavaScript -->
+    <!-- jQuery first, then Popper.js, then Bootstrap JS -->
```

Comment:
Also good job with leaving this comments so myself and any future developer (like future you) can more quickly and easily understand what everything is for. Great habit to maintain